Good-Bids/goodbids
# Dynamic Bid Pricing

# Summary

This PR implements the Bid auto-increment feature when a bid is successfully placed on an Auction, the Bid Product price will increment by the Bid Increment amount, ready for the next bid.

It also includes a Meta Box on the Order Detail page that shows the Related Auction and Reward Product details.

## Issues

* #25 

## Testing Instructions

1. Create a Reward Product (if you have not already done so)
2. Set up WooCommerce Stripe (if you have not already done so)
3. Create a new Auction - Even if you already have an existing auction.
4. Visit the Auction (you should see a Bid Now button)
5. Click the Bid Now button
6. Complete the checkout
7. You should be redirected back to the Auction
8. The Bid Now should be updated with the new Bid amount
9. Clicking "Bid Now" should add the Bid to your cart with the correct, updated Bid Amount

## Screenshots

**Order Auction Info Meta Box**
<img width="296" alt="Screenshot 2023-12-18 at 11 37 24 AM" src="https://github.com/Good-Bids/goodbids/assets/122309362/4dca0d55-4f02-4dfd-8f67-aa37e1699939">
